src/model.js

```diff
@@ -46,4 +47,28 @@ Model.prototype.frameGeometry = function (frameIndex) {
   };
 };
 
+Model.prototype.stepFrames = function (step) {
+  step = Math.max(step, 0);
+  step = Math.min(step, this.steps - 1);
+  return [step * this.fps, step * this.fps + this.fps - 1];
+};
+
+Model.prototype.stepFromFrame = function (frame) {
+  return Math.floor(frame / this.fps);
+};
+
+Object.defineProperty(Model.prototype, 'steps', {
+  enumerable: true,
+  get: function () {
+    return this.frames.length / this.fps;
+  }
+});
+
+Object.defineProperty(Model.prototype, 'lastFrame', {
+  enumerable: true,
+  get: function () {
+    return this.frames.length - 1;
+  }
+});
```

Comment:
So, I'm trying this out...I'm not sure I like this style more than defining them individually.